scripts/testnet/sovereignBridge/config/token.snippets.sh

```diff
@@ -1,9 +1,11 @@
 issueToken() {
     manualUpdateConfigFile #update config file
 
-    local TOKENS_TO_MINT=$(echo "scale=0; $INITIAL_SUPPLY*10^$NR_DECIMALS/1" | bc)
+    echo "Issuing fungible token on main chain..."
 
-    mxpy --verbose contract call ${ESDT_SYSTEM_SC_ADDRESS} \
+    local TOKENS_TO_MINT=$(echo "scale=0; $INITIAL_SUPPLY*10^$NR_DECIMALS/1" | bc)
```

Comment:
Is this multiplication safe for very large numbers?

Reply:
ChatGPT: `By default, bc supports up to approximately 9223372036854775807 (2^63 - 1) for integers on many systems`.